packages/twenty-front/src/pages/settings/developers/api-keys/SettingsDevelopersApiKeysNew.tsx

```diff
@@ -64,23 +63,27 @@ export const SettingsDevelopersApiKeysNew = () => {
   };
   const canSave = !!formValues.name && createOneApiKey;
   return (
-    <SubMenuTopBarContainer Icon={IconSettings} title="Settings">
+    <SubMenuTopBarContainer
+      Icon={IconCode}
+      title={
+        <Breadcrumb
```

Comment:
This transclusion pattern is not common in the codebase and makes the code harder to read